qa/L0_tf_gpu_io/test.sh

```diff
@@ -42,6 +42,7 @@ export CUDA_VISIBLE_DEVICES=0
 
 CLIENT=../clients/perf_client
 BACKENDS=${BACKENDS:="graphdef savedmodel"}
+MODEL_TYPES="$BACKENDS mismatch_key_name"
```

Comment:
Should just make it an additional case in this test so you don't need to have the additional type check and if-else below

Reply:
Sure.